packages/typespec-ts/test/util/emitUtil.ts

```diff
@@ -440,7 +442,14 @@ export async function emitModularOperationsFromTypeSpec(
         modularCodeModel.clients[0],
         dpgContext,
         modularCodeModel,
-        false
+        false,
+        env["EXPERIMENTAL_TYPESPEC_TS_SERIALIZATION"]
```

Comment:
do we have to set the env EXPERIMENTAL_TYPESPEC_TS_SERIALIZATION for the serialize utils unit tests? curious if those tests are really testing something now?

Reply:
The environment does have to be set for unit tests to consume the new code. We have expected test failures in this PR, so as tests start to pass we can look into selectively enabling it for those tests.